### ISSUE TITLE
Shuffling formations

### ISSUE THREAD

**Issue by gdwitt** [LordGolias/Brytenwalda#22]
When groups form ranks or shieldwall, they starting moving at an angle towards the enemy even when I give the order "hold this position."
The only way I have to get them to stop is to say break formation or to tell them to move backwards 10 paces.
This is a distressing bug and I'm not sure the AI has the same problem.
My testing also shows that player parties in formation do very poorly versus enemies in formation or not in formation. They do very bad against enemies not in formation. Formations need a big boost somehow. 
Motomatura was the developer of the formation system and he has been very busy with the Viking Conquest project. I think he is an American programmer that is busy with a new project now (we communicated on Reddit). This was not an issue back in 2013 when I first started playing. If we have too little time for formations I would:
1) compare to 2013 Brytenwalda
2) Ask for help from other Brytenwalda mods and use their formation code. Produno and Idibil might be able to help.

**Comment by HooTmAnTW** [comment]
Yes, i have this problem, too. I think it is kind of an older problem (repolished), i am not shure. 

Why is it closed? Did you set it to 2013 BW, or could they help you ? just asking because this bug did bother me many times.

**Comment by LordGolias** [comment]
It is not closed. The closed one is #23, which was a duplicated of this one. I also have seen this problem. I actually started working on a solution, but I still haven't finished it.

**Comment by HooTmAnTW** [comment]
Take your time i will take a look too. but my phyton skills are still very low ^^ i was also looking through the net like 1 h about this, could not find anything usefull about the bug or the formation codeing part.

**Comment by HooTmAnTW** [comment]
for me it works better now on version 24 patch A.

**Comment by gdwitt** [comment]
The only thing I touched in the patch is adding Somebody's toggle weapon script.
In my two tests, the formations are manageable if one uses a move forward or backwards command.
The default formation causes shuffling. I could turn formations off, but that means all battles will devolve into the AI being penalized with their troops rushing forward into combat without allies around.
